esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2375,6 +2375,9 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
         {
             RemoteFilename rfn;
             SocketEndpoint ep(destip);
+            if (ep.isNull())
+                throw MakeStringException(ECLWATCH_INVALID_INPUT, "Cannot resolve destination network IP from %s.", destip);
```

Comment:
It might be unclear what it was doing when it hit this error.
So probably the message should mention that it is trying to despray.